addons/pos_loyalty/static/src/overrides/components/product_screen/order_summary/order_summary.js

```diff
@@ -47,6 +50,23 @@ patch(OrderSummary.prototype, {
                 return;
             }
         }
+        const numpadMode = this.pos.numpadMode;
+        if (
+            selectedLine?.uiState?.program_type === "gift_card" &&
+            ["quantity", "discount"].includes(numpadMode) &&
+            (!Number.isNaN(Number(key)) || key === ".")
```

Comment:
```suggestion
            key !== 'Backspace'
```

Is that possible to do something like that ? Since the only action you can do is delete the line ?

Reply:
Hello @davidmonnom 

Changes are done.

Thanks!